### PR TITLE
Feature/lior/alerts

### PR DESCRIPTION
**Pull request by LiorRacer**
**What this PR does / why we need it**:
Alerts support added to BigQuery plugin
**Which issue(s) this PR fixes**:

Fixes # https://github.com/doitintl/bigquery-grafana/issues/176

**Special notes for your reviewer**:

**Release note**:

```release-note
Alerts for BigQuery plugin 
```

### CODE REVIEW

File: dist/module.js [hunk 1]
```diff
@@ -59623,6 +59623,9 @@ var _response_parser2 = _interopRequireDefault(_response_parser);
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
 
+// import { validate } from "@babel/types";
```
Comment (Tamirk) [inline dist/module.js:59626]: Can we remove this comments lines?

File: pkg/datasource.go [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+	"golang.org/x/net/context/ctxhttp"
+)
+
+type BQDatasource struct {
+	plugin.NetRPCUnsupportedPlugin
+	logger hclog.Logger
+}
+
+func (t *BQDatasource) Query(ctx context.Context, tsdbReq *datasource.DatasourceRequest) (*datasource.DatasourceResponse, error) {
+	t.logger.Debug("Query", "datasource", tsdbReq.Datasource.Name, "TimeRange", tsdbReq.TimeRange)
```
Comment (Tamirk) [inline pkg/datasource.go:27]: Where can we see the debug messages after code is deployed?

File: pkg/datasource.go [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+	switch remoteDsReq.queryType {
+	case "search":
+		return t.parseSearchResponse(body)
+	default:
+		return t.parseQueryResponse(remoteDsReq.queries, body)
+	}
+}
+
+func (t *BQDatasource) createRequest(tsdbReq *datasource.DatasourceRequest) (*remoteDatasourceRequest, error) {
```
Comment (Tamirk) [inline pkg/datasource.go:57]: Missing comments in this method

File: pkg/datasource.go [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+				Rows:    make([]*datasource.TableRow, 0),
+			}
+
+			for _, c := range r.Columns {
+				table.Columns = append(table.Columns, &datasource.TableColumn{
+					Name: c.Text,
+				})
+			}
+
+			for _, row := range r.Rows {
```
Comment (Tamirk) [inline pkg/datasource.go:139]: What's the puspose of this `for` add a comment on this

File: pkg/datasource.go [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+			&datasource.TableColumn{Name: "text"},
+		},
+		Rows: make([]*datasource.TableRow, 0),
+	}
+
+	for n := 0; n < metricCount; n++ {
+		values := make([]*datasource.RowValue, 0)
+		jm := jBody.GetIndex(n)
+
+		if text, found := jm.CheckGet("text"); found {
```
Comment (Tamirk) [inline pkg/datasource.go:209]: Can we have enum for the "text", "value" and other Strings in this code snippet

File: pkg/main.go [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+	"os"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend/datasource"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
+)
+
+func main() {
+	log.DefaultLogger.Info("lior test")
```
Comment (Tamirk) [inline pkg/main.go:11]: Remove this log before production

File: pkg/plugin.go [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+		data.NewField("values", nil, []int64{10, 20}),
+	)
+
+	// add the frames to the response
+	response.Frames = append(response.Frames, frame)
+
+	return response
+}
+
+// CheckHealth handles health checks sent from Grafana to the plugin.
```
Comment (Tamirk) [inline pkg/plugin.go:103]: 👍

File: dist/module.js (hunk 1 above)
Comment (LiorRacer) [reply dist/module.js:59626]: Sure

File: pkg/datasource.go (hunk 2 above)
Comment (LiorRacer) [reply pkg/datasource.go:27]: The logs are at grafana.log file

**Pull request merged**